gu-base no longer builds on Rust 1.36+

```
   Compiling gu-actix v0.1.0 (https://github.com/golemfactory/golem-unlimited?branch=feature/gu-client-rpc#36cb510e)
   Compiling gu-base v0.1.0 (https://github.com/golemfactory/golem-unlimited?branch=feature/gu-client-rpc#36cb510e)
   Compiling gu-net v0.1.0 (https://github.com/golemfactory/golem-unlimited?branch=feature/gu-client-rpc#36cb510e)
error[E0283]: type annotations required: cannot resolve `std::string::String: std::convert::AsRef<_>`
   --> /home/marcin/.cargo/git/checkouts/golem-unlimited-8aa5a0012b75d98f/36cb510/gu-base/src/output.rs:114:30
    |
114 |             let prg = self.0.as_ref();
    |                              ^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0283`.
error: Could not compile `gu-base`.
warning: build failed, waiting for other jobs to finish...
error: build failed
marcin@archtop /home/marcin/proj/golem/gumpi [101] (guclient|✚1)
[10:44:28] $ rustup run nightly cargo --version
cargo 1.37.0-nightly (545f35425 2019-05-23)
```
This also happens on beta:
```
cargo 1.36.0-beta (c4fcfb725 2019-05-15)
```

--- a/gu-actix/src/pipe.rs
+++ b/gu-actix/src/pipe.rs
@@ -132,7 +132,6 @@ impl<T, E> futures::Sink for AsyncWriter<T, E> {
 
 impl io::Write for Writer<Bytes, io::Error> {
     fn write(&mut self, buf: &[u8]) -> Result<usize, io::Error> {
-        use bytes::Bytes;
         self.send(Ok(Bytes::from(buf)))
             .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
         Ok(buf.len())

--- a/gu-base/src/cli.rs
+++ b/gu-base/src/cli.rs
@@ -33,7 +33,6 @@ where
     use std::collections::hash_map::RandomState;
     use std::collections::HashMap;
     use std::iter::FromIterator;
-    use std::iter::IntoIterator;
 
     let mut vec = Vec::new();
     for row in it {

--- a/gu-base/src/output.rs
+++ b/gu-base/src/output.rs
@@ -111,7 +111,7 @@ impl Module for AutocompleteModule {
 
         if let Some(sub_matches) = matches.subcommand_matches("completions") {
             let shell = sub_matches.value_of("SHELL").unwrap();
-            let prg = self.0.as_ref();
+            let prg: &str = self.0.as_ref();
             app_gen().gen_completions_to(prg, shell.parse().unwrap(), &mut io::stdout());
             return true;
         }

--- a/gu-base/src/output.rs
+++ b/gu-base/src/output.rs
@@ -111,7 +111,7 @@ impl Module for AutocompleteModule {
 
         if let Some(sub_matches) = matches.subcommand_matches("completions") {
             let shell = sub_matches.value_of("SHELL").unwrap();
-            let prg : &str = self.0.as_ref();
+            let prg: &str = self.0.as_ref();
             app_gen().gen_completions_to(prg, shell.parse().unwrap(), &mut io::stdout());
             return true;
         }

--- a/gu-base/src/output.rs
+++ b/gu-base/src/output.rs
@@ -111,7 +111,7 @@ impl Module for AutocompleteModule {
 
         if let Some(sub_matches) = matches.subcommand_matches("completions") {
             let shell = sub_matches.value_of("SHELL").unwrap();
-            let prg: &str = self.0.as_ref();
+            let prg : &str = self.0.as_ref();
             app_gen().gen_completions_to(prg, shell.parse().unwrap(), &mut io::stdout());
             return true;
         }

--- a/gu-lan/src/actor.rs
+++ b/gu-lan/src/actor.rs
@@ -246,7 +246,6 @@ impl<T: MdnsConnection> Actor for MdnsActor<T> {
 
     /// Creates stream handler for incoming mDNS packets
     fn started(&mut self, ctx: &mut Self::Context) {
-
         let socket = Self::create_mdns_socket().expect("Creation of mDNS socket failed");
         let (sink, stream) = UdpFramed::new(socket, MdnsCodec(T::unicast_query())).split();
 

--- a/gu-lan/src/actor.rs
+++ b/gu-lan/src/actor.rs
@@ -246,7 +246,6 @@ impl<T: MdnsConnection> Actor for MdnsActor<T> {
 
     /// Creates stream handler for incoming mDNS packets
     fn started(&mut self, ctx: &mut Self::Context) {
-        use futures::Stream;
 
         let socket = Self::create_mdns_socket().expect("Creation of mDNS socket failed");
         let (sink, stream) = UdpFramed::new(socket, MdnsCodec(T::unicast_query())).split();

--- a/gu-lan/src/module.rs
+++ b/gu-lan/src/module.rs
@@ -42,7 +42,7 @@ pub fn format_instances_table(instances: &HashSet<ServiceInstance>) {
 }
 
 fn run_client(instances: &String) {
-    use actix::{self, SystemService};
+    use actix::SystemService;
 
     let sys = actix::System::new("gu-lan");
     let instances = instances.split(',').map(|s| s.to_string().into()).collect();

